content/docs/command-reference/queue/start.md

```diff
@@ -0,0 +1,54 @@
+## queue start
+
+Start the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue worker process.
+
+## Synopsis
+
+```usage
+usage: dvc queue start [-h] [-q | -v] [-j <number>]
+```
+
+## Description
+
+Starts the task queue worker process. The worker process will consume and
+execute one queued experiment task at a time in the background, until either
+`dvc queue stop` is used or the queue is empty.
+
+> For performance reasons, when the queue is empty, the worker will idle for up
+> to 60 seconds before exiting. If new experiment tasks are added to the queue
+> during this time, the worker will resume processing tasks. Otherwise, the
+> worker will exit after the 60 second idle timeout.
+
+Queued tasks are run sequentially by default, but can be run in parallel using
+the `--jobs` option.
+
+> ⚠️ Parallel runs are experimental and may be unstable. Make sure you're using
+> a number of jobs that your environment can handle (no more than the CPU
+> cores).
+
+> Note that since queued experiments are run isolated from each other, common
+> stages may sometimes be executed several times depending on the state of the
+> [run-cache] at that time.
+
+## Options
+
+- `-j <number>`, `--<number>` - start up to this number of workers in parallel.
+  Defaults to 1 (the task queue is processed serially).
+
+  > Note that if any queue worker processes have already been started, this
+  > command will not start additional processes unless `number` is greater than
+  > the number of existing workers (`number` is treated as the maximum allowed
+  > concurrency value).
```

Comment:
👀 So it's possible to do `dvc queue start` followed by `dvc queue start -j 2` now? I didn't realize this was already supported.

Reply:
Yes this is enabled now

Reply:
> 👀 So it's possible to do `dvc queue start` followed by `dvc queue start -j 2` now? I didn't realize this was already supported.

It will show that only create 1 new worker at the second `queue start`